demo/label_semantic_roles/train_ft.py

```diff
@@ -0,0 +1,229 @@
+import os
+import math
+import numpy as np
+import paddle.v2 as paddle
+import paddle.v2.dataset.conll05 as conll05
+import paddle.v2.evaluator as evaluator
+from paddle.v2.reader.creator import cloud_reader
+
+master_ip = os.getenv("MASTER_IP")
```

Comment:
Since #278 has already merged, we can use environment variable `ETCD_IP` instead of `MASTER_IP`.